includes/Core/Consent_Mode/REST_Consent_Mode_Controller.php

```diff
@@ -150,10 +150,11 @@ protected function get_rest_routes() {
 					array(
 						'methods'             => WP_REST_Server::READABLE,
 						'callback'            => function () {
-							$is_active = function_exists( 'wp_set_consent' );
-							$installed = $is_active;
-							$slug      = 'wp-consent-api';
-							$plugin    = "$slug/$slug.php";
+							$is_active  = function_exists( 'wp_set_consent' );
+							$installed  = $is_active;
+							$plugin_uri = 'https://wordpress.org/plugins/wp-consent-api';
+							$slug       = 'wp-consent-api';
```

Comment:
Let's inline slug since this value should never change for the important parts where it's used (`$install_url`). It can also be inlined in the initial value for `$plugin`.

Reply:
Updated, thank you!